src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -143,6 +145,7 @@ class BasePaymentsPage extends React.Component {
                 };
             }
             this.setState({
+                isSelectedPaymentMethodDefault: !accountDefault,
```

Comment:
Don't you think that variable does not justify the value it stores? Shouldn't it be storing the reversed value?
```suggestion
                isSelectedPaymentMethodDefault: accountDefault,
```